ofl/kedebideri/OFL.txt

```diff
@@ -3,7 +3,8 @@ with Reserved Font Names "Kedebideri" and "SIL".
 Copyright (c) 2010-2024 Adobe (https://www.adobe.com/),
 with Reserved Font Name "Source" (Latin glyphs).
 
-This license is copied below, and is also available with an FAQ at:
+This Font Software is licensed under the SIL Open Font License, Version 1.1,

```

Comment:
The final character is a comma not a period which is probably a fatal error also